src/tristan/command_line/images.py

```diff
@@ -12,7 +12,11 @@
 from dask import array as da
 from dask.diagnostics import ProgressBar
 from hdf5plugin import Bitshuffle
-from nexgen.copy import CopyTristanNexus
+
+try:
+    from nexgen.copy import CopyTristanNexus
+except ModuleNotFoundError:
+    from nexgen.nxs_copy import CopyTristanNexus  # nexgen version >= 0.4.8
```

Comment:
Since the structure of Nexgen will include `nexgen.nxs_copy` for the foreseeable future and `nexgen.copy` is the the legacy structure, I would suggest the `except` statement ought to encode the fallback, i.e. the legacy version.  In other words, I suggest flipping the order of the imports in the `try` and `except` statements.

```suggestion
try:
    from nexgen.nxs_copy import CopyTristanNexus  # nexgen version >= 0.4.8
except ModuleNotFoundError:
    from nexgen.copy import CopyTristanNexus
```

Reply:
Makes sense